web/src/components/app.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from 'react';
-import { Provider } from 'react-redux';
-import Pages from './pages';
-import { isMobileWebkit, isFocus, isNativeIOS, sleep } from '../utility';
+import { connect, Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
+const { LocalizationProvider } = require('fluent-react');
```

Comment:
Why `require()`? `import {LocalizationProvider} from 'fluent-react/compat';` should work, too.

Reply:
That's a TS thing, as module resolution works (by default) differently from CommonJS. There's a flag to enable interop, but that has some ramifications we didn't talk about yet.